src/styles/main.scss

```diff
@@ -0,0 +1,151 @@
+@import "./variables";
+@import "./reset";
+@import "./animation-stepOn60";
+
+%abs100 {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+}
+
+body {
+  background: conic-gradient(
+    rgba(255, 255, 255, 0.8),
+    rgba(0, 0, 0, 0.2),
+    rgba(255, 255, 255, 0.8)
+  );
+  height: 100vh;
+  margin: 0;
+  display: flex;
+  justify-content: center;
+}
+
+.watch {
+
+  & > * {
+    opacity: 0.68;
+  }
+
+  margin: auto;
+  width: $size-main;
+  height: $size-main;
+
+  position: relative;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  border-radius: 50%;
+  border-style: dotted;
+  border-color: $c-circles;
+
+  border-width: $size-circle;
+
+  &__pad {
+    opacity: 0.5;
+    position: absolute;
+
+    width: $size-pad;
+    height: $size-pad;
+
+    &-lines {
+      stroke: $c-lines;
+      stroke-width: 3px;
+    }
+  }
+
+  &::after {
+    content: "";
+
+    @extend %abs100;
+
+    border-radius: 50%;
+  }
+
+  &__seconds {
+
+    width: $size-second;
+    height: $size-second;
+
+    position: absolute;
+    z-index: 3;
+
+    &-second {
+
+      @extend %abs100;
+
+      animation-name: stepOn60;
+      animation-duration: 60s;
+      animation-iteration-count: infinite;
+
+      &::before {
+        border-radius: $size-br-radius;
+
+        content: "";
+        position: absolute;
+        top: 0;
+        left: 49%;
+        width: 2%;
+        height: 50%;
+        background: $c-seconds;
+        box-shadow: 0 0 $size-glow $size-glow-blur $c-sec-glow;
+      }
+
+      &::after {
+        content: "";
+        position: absolute;
+        top: 48%;
+        left: 48%;
+        width: 4%;
+        height: 4%;
```

Comment:
where does these values come from?

Reply:
its a dot that symbolize axis of watch hand
to place it centered - I used 50% - 1/2 of its own height
I decided to not put this in formula to code not to be overcomplicated